zh/upgrade-tidb-operator.md

```diff
@@ -5,17 +5,39 @@ category: how-to
 
 # 升级 TiDB Operator
 
-本文介绍如何升级 TiDB Operator。升级 TiDB Operator 和自定义 TiDB Operator 类似，修改 `values.yaml` 中的镜像版本，然后执行 `helm upgrade`：
+本文介绍如何升级 TiDB Operator。
```

Comment:
I think we should point out the risks for upgrading operator? For exmaple, whether it would cause tidbcluster rolling updated.

Reply:
@DanielZhangQD PTAL😀

Reply:
I have added the upgrading of discovery, other components will not be upgraded.
For the other effects of changing from chart to CR, I will add a new doc.